wot-wg-2023-details.html

```diff
@@ -132,53 +132,53 @@ <h1>Work Item Details for Web of Things Working Group 2023 Charter Draft</h1>
           <dt>
             <a href="#http-binding-workitem"><strong>HTTP Protocol Binding</strong></a> (Binding Templates):
```

Comment:
why is always "(Binding Templates)" provided everywhere?

Reply:
In the initial template, each work item had the deliverable it was concerned about next to it in parentheses. Other items in this list have it still (https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html). Given that we will not have a deliverable called binding templates, maybe we should remove this or change it to TD?

Reply:
I thought I removed all those deliverable assignments but I guess I forgot some.  At any rate, I'm ok with this PR, although we're going to probably reorganize all this shortly.